src/test/java/pl/coderslab/charity/service/CategoryServiceTest.java

```diff
@@ -0,0 +1,70 @@
+package pl.coderslab.charity.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import pl.coderslab.charity.entity.Category;
+import pl.coderslab.charity.repository.CategoryRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class CategoryServiceTest {
+
+    @Mock
+    private CategoryRepository categoryRepository;
+
+    @InjectMocks
+    private CategoryService categoryService;
+
+    @Test
+    void getAllCategoriesShouldReturnCategoryList() {
+        when(categoryRepository.findAll()).thenReturn(List.of(new Category(1L, "test"), new Category(2L, null)));
```

Comment:
List.of(new Category(1L, "test"), new Category(2L, null)) to najlepiej wrzucić do prywatnej metody 
private List<Category> getCategories(){
return List.of(new Category(1L, "test"), new Category(2L, null));
}